align `pynxtools.nomad` module with recommended nomad plugin structure (#732)
FAIRmat-NFDI/pynxtools: src/pynxtools/nomad/example_uploads/iv_temp_example/README.md

# Introduction

> This is an example of a PID controlled sensor sweep scan. The temperature is set using a PID controller. Then, for the set temperature, a voltage sweep is performed. For each of the voltages, the current is measured. This is repeated for a given list of temperatures.

Reviewer: ```suggestion
This is an example of a PID controlled sensor sweep scan. The temperature is set using a PID controller. Multiple experiments at different temperatures are performed. For each, a voltage sweep is performed, recording voltage over current.
```

> This specific data was captured using a Bluesky controlled system. This was then saved into a binary file using Pickle. This example illustrates how such a dataset could be converted using the [`JSONMapReader`](https://github.com/FAIRmat-NFDI/pynxtools/tree/master/src/pynxtools/dataconverter/readers/json_map) of the NOMAD plugin [`pynxtools`](https://github.com/FAIRmat-NFDI/pynxtools).

Reviewer: ```suggestion
This specific data was captured using a [Bluesky](https://blueskyproject.io/) controlled system, writing to a binary file using Pickle. This example illustrates how such a dataset could be converted using the [`JSONMapReader`](https://github.com/FAIRmat-NFDI/pynxtools/tree/master/src/pynxtools/dataconverter/readers/json_map) of the NOMAD plugin [`pynxtools`](https://github.com/FAIRmat-NFDI/pynxtools).
```

You may also view your supplied or generated NeXus files here with the H5Web viewer.
To do so open the **FILES** tab and just select a `.nxs` file.

## Using a Jupyter Notebook

> This example comes with a very simple Jupyter Notebook that shows how one could easily get access to a Python environment with access to all your data in one place.

Reviewer: ```suggestion
This example comes with a simple Jupyter Notebook that shows how one could easily get access to a Python environment with access to all your data in one place.
```

> To give this a go, click the **FILES** tab and select `IV_temp.ipynb`. Feel free to modify this or just create a new one to try!

Reviewer: ```suggestion
To give this a go, click the **FILES** tab and select `IV_temp.ipynb`. Feel free to modify this notebook or just create a new one to try!
```
